ubidots/ubidots_in.html

```diff
@@ -50,78 +50,78 @@
       </div>
       <!-- VARIABLE 1 -->
       <div id="variable-1" class="form-row" style="margin-bottom:10px">
-        <label style="width:24px" id=node-input-variable-1-space></label>
-         <input type="text" style="width: calc(100% - 250px); margin-left:28px" id="node-input-label_variable_1" />
+         <label style="width:23px" id=node-input-variable-1-space></label>
+         <input type="text" style="width: calc(100% - 250px)" id="node-input-label_variable_1" />
```

Comment:
With the flexbox container you could only do `flex: 1` here and forget about calculating the width that way.